README.md

```diff
@@ -144,6 +144,8 @@ Here is a list of community roles with current and previous members:
    - [Paulo Janotti](https://github.com/pjanotti), Splunk
 
 Learn more about roles in [Community membership](https://github.com/open-telemetry/community/blob/main/community-membership.md).
+In addition to what is described at the organization-level, the SIG Collector requires all core approvers to take part in rotating
```

Comment:
Does this assume that all core approvers are also contrib approvers?  Should that be made explicit?

Reply:
I think it assumes, but I don't think it should be.

Reply:
The idea is that a specific group should be responsible for the releases, and the people belonging to this group should rotate doing this work. If a core approver isn't an approver on contrib, they should request assistance from a cotnrib approver. I'll make this explicit.

If you think another group should be responsible for the release, let me know.